manifests/pipeline.yaml

```diff
@@ -35,7 +35,13 @@ parameters:
     value: quay.io/coreos-assembler/coreos-assembler:main
   - description: AWS S3 bucket in which to store builds (or blank for none)
     name: S3_BUCKET
-    value: fcos-builds
+    value: ''
+  - description: AWS S3 principle to be used (or blank for none)
+    name: S3_PRINCIPLE
+    value: ''
+  - description: AWS S3 profile to be used (or blank for none)
+    name: S3_PROFILE
+    value: ''
```

Comment:
Shouldn't those be e.g. `BREW_PRINCIPAL` and `BREW_PROFILE`?

Reply:
Yeah, fixing